src/System.Globalization/tests/Performance/Perf.CompareInfo.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
```

Comment:
>Performance [](start = 22, length = 11)

I suggest in these tests we test with some longer strings too.

Reply:
Please include couple of computationally hard cases, i.e.:
2000x'A', 50000x'A', 1x'B'
search for: 50000x'A', 1x'B'
(replace A/B with different kinds of letters: non-ASCII, surrogates, similarly as you are already doing)